docs/openstack/mariadb_copy.md

```diff
@@ -112,3 +112,18 @@ SOURCE_MARIADB_IP=127.17.0.100
   oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
      mysql -h "$PODIFIED_MARIADB_IP" -uroot "-p$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;'
   ```
+
+* During the pre/post checks the pod `mariadb-client` might have returned a pod security warning
+  related to the `restricted:latest` security context constraint. This is due to default security
+  context constraints and will not prevent pod creation by the admission controller. Given the
+  pod security admission controller doesn't directly interfer with the short lived pod to solve you can:
+
+    - For most workloads, the `restricted` SCC limits pod privileges
+      adequately. When creating a workload, you receive the warning,
+      but the workload creates pods and works correctly.
+
+    - You can also create/run your workloads by providing a full specification
+      of the security constraints to apply, without relying on the SCC to
+      provide the configuration. By following this approach, privileges are
+      more explicit, and you have greater control, so you can adjust more
+      precisely the specific privileges that are granted to workloads.
```

Comment:
I would probably drop this second part as it does not apply to the MariaDB copy here. (If it did, we should probably just fix the bash snippets instead of documenting the warning.)

So the information could be a single paragraph where the gist is "You'll see a warning for the short-lived pod but it will not interfere with functionality." And for more info we could perhaps just link the article you've found, to keep the docs succinct. ( https://learn.redhat.com/t5/DO280-Red-Hat-OpenShift/About-pod-security-standards-and-warnings/m-p/32502 )